dev-system/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:7
+
+RUN yum install -y java-1.8.0-openjdk
```

Comment:
We should be using Oracle Java here as well.

Reply:
Done